boards/arm/dragonclaw/Kconfig.board

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022 Google Inc
+# SPDX-License-Identifier: Apache-2.0
+
+# Dragonclaw is a board created by Google for fingerprint-related functionality
+# development. Board schematics, layout and BOM is available at:
+# https://chromium.googlesource.com/chromiumos/platform/ec/+/HEAD/docs/schematics/dragonclaw
```

Comment:
This should go in the board documentation file (see Erwan comment)

Reply:
Fixed. Thanks!